gnomad/sample_qc/relatedness.py

```diff
@@ -681,7 +793,7 @@ def compute_related_samples_to_drop(
             "Computing samples related to too many individuals (>%d) for exclusion",
             min_related_hard_filter,
         )
-        gbi = relatedness_ht.annotate(s=list(relatedness_ht.key))
+        gbi = relatedness_ht.annotate(s=[relatedness_ht.key[0], relatedness_ht.key[1]])
```

Comment:
Is this change for readability or does casting no longer work?

Reply:
Or I guess speed?

Reply:
`list(relatedness_ht.key)` returns ['i', 'j'] for each `s`, I'm assuming this was something that changed and was not always the case, but I couldn't confirm.